tauri-app/src-tauri/src/commands/fork.rs

```diff
@@ -0,0 +1,13 @@
+use rain_orderbook_common::fork::fork_parse_dotrain;
+
+#[tauri::command]
+pub async fn fork_parse(
+    rainlang: &str,
+    front_matter: &str,
+    fork_url: &str,
+    fork_block_number: u64,
+) -> Result<(), String> {
```

Comment:
can this return a `CommandResult` from https://github.com/rainlanguage/rain.orderbook/blob/main/tauri-app/src-tauri/src/error.rs

Reply:
yes it can, I asked in the chat if you guys need it to return the expconfig (the bytes you asking for here), which everyone said no, so I skipped it

Reply:
this can return anything for the Ok variant, but text editor wont use it as we discussed, but feel free to change the Ok variant to wahtever you'd like